lib/kino/process.ex

```diff
@@ -421,15 +423,36 @@ defmodule Kino.Process do
     {func_result, sequence_diagram}
   end
 
-  defp generate_participant_entry(pid, idx) do
+  # :proc_lib.get_label/1 was added in OTP 27
```

Comment:
Same in other places

```suggestion
  # TODO: use :proc_lib.get_label/1 once we require OTP 27
```